playwright/element-web-test.ts

```diff
@@ -316,7 +316,6 @@ export const expect = baseExpect.extend({
         name?: `${string}.png`,
         options?: {
             mask?: Array<Locator>;
-            omitBackground?: boolean;
```

Comment:
This is an option supported by the underlying playwright function, so I'm not sure that ripping it out here is necessary. But if it doesn't work for us, I guess it makes sense.

Reply:
Good spot